cmd/charm/charmcmd/cmd.go

```diff
@@ -181,7 +181,9 @@ func addAuthFlag(f *gnuflag.FlagSet, s *string) {
 
 // addChannelFlag adds the -c (--channel) flags to the given flag set.
 func addChannelFlag(f *gnuflag.FlagSet, s *string) {
-	f.StringVar(s, "c", string(params.StableChannel), "the channel the charm or bundle is assigned to")
+	f.StringVar(s, "c",
+		fmt.Sprint(string(params.StableChannel), `|`, string(params.DevelopmentChannel)),
+		"the channel the charm or bundle is assigned to")
 	f.StringVar(s, "channel", string(params.StableChannel), "")
```

Comment:
Not your fault, but there's no point in specifying the default here. Both the default and description are taken from the shortest flag.